src/anaconda/install.sh

```diff
@@ -103,4 +103,25 @@ if ! conda --version &> /dev/null ; then
     updaterc "export CONDA_DIR=${CONDA_DIR}/bin"
 fi
 
+# Display a notice on conda when not running in GitHub Codespaces
+mkdir -p /usr/local/etc/vscode-dev-containers
+cat << 'EOF' > /usr/local/etc/vscode-dev-containers/conda-notice.txt
+When using "conda" from outside of GitHub Codespaces, note the Anaconda repository contains
+restrictions on commercial use that may impact certain organizations. See https://aka.ms/ghcs-conda
+
+EOF
+
+notice_script="$(cat << 'EOF'
+if [ -t 1 ] && [ "${IGNORE_NOTICE}" != "true" ] && [ "${TERM_PROGRAM}" = "vscode" ] && [ "${CODESPACES}" != "true" ] && [ ! -f "$HOME/.config/vscode-dev-containers/conda-notice-already-displayed" ]; then
+    cat "/usr/local/etc/vscode-dev-containers/conda-notice.txt"
+    mkdir -p "$HOME/.config/vscode-dev-containers"
+    ((sleep 10s; touch "$HOME/.config/vscode-dev-containers/conda-notice-already-displayed") &)
+fi
+EOF
+)"
+
+if [ -f "/etc/zsh/zshrc" ]; then
```

Comment:
Wouldn't this cause the `/etc/bash.bashrc` file to not be written if ZSH isn't installed? It seems like we should split the two out and have one conditional for bash and another conditional for ZSH -- each would only install if the respective shell is installed.

Reply:
Good catch! Addressed that.